scripts/aarch64-libc2_17-cross-build.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+## NOTE:
+## This script must be run in the centos:7 container which is similar to "manylinux2014_aarch64"
+## and has the same toolchain version / glibc / kernel / ABI.
+
+set -e
+
+# install required cross compiler/linker
+yum install -y dnf epel-release gcc curl
+yum install -y binutils-aarch64-linux-gnu gcc-aarch64-linux-gnu gcc-c++-aarch64-linux-gnu
+
+# install Rust toolchain
+curl https://sh.rustup.rs -sSf | sh -s -- -y
+source "$HOME/.cargo/env"
+# install our target
+rustup target add aarch64-unknown-linux-gnu
+
+# since the Centos is quite old, and were are on the different architecture, we will disable gpg verification for now
+sed -i s/gpgcheck=1/gpgcheck=0/g /etc/yum.repos.d/CentOS-*
+# install chroot with some of the deps we will need for cross compilation
+dnf --forcearch aarch64 --release 7 install gcc glibc glibc-devel --installroot /usr/aarch64-linux-gnu/sys-root/ -y
+
+# FIX: linker cannot find the libgcc_s, since we have libgcc_s.so.1, so we just link it
+ln -s /usr/aarch64-linux-gnu/sys-root/usr/lib64/libgcc_s.so.1 /usr/aarch64-linux-gnu/sys-root/usr/lib64/libgcc_s.so
```

Comment:
If you put all of the steps up until here into a dockerfile, we could benefit from caching these steps.

Reply:
Yeah, agree - that's what I mentioned in the description, as a followup for the improvements.  
This PR was just to show the improvement we can gain and if those look are ok.  

I will re-work this script and will use `Dockerfile` for all of this, including installing python dependencies to include building the wheel in one go. And also will add caching for built docker image.

Reply:
If you look at the current builder image for the main relay binary, you'll notice that it's based on `centos:7` as well. It does not perform a cross build just now, but you might be able to reuse this and just install a cross toolchain in there.

https://github.com/getsentry/relay/blob/984717e894b84570903a2901036212ca559acf47/Dockerfile#L8